packages/edit-post/src/editor.js

```diff
@@ -126,7 +126,7 @@ function Editor( {
 		};
 
 		// Omit hidden block types if exists and non-empty.
-		if ( size( hiddenBlockTypes ) > 0 ) {
+		if ( hiddenBlockTypes.length > 0 ) {
```

Comment:
I can see some code references [like this](https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-post/src/store/actions.js#L319) that also check if `hiddenBlockTypes` is not null/undefined, which lodash [handled](https://github.com/lodash/lodash/blob/master/size.js#L30). 

By quickly checking the code. I think it's just defensive programming and we always fallback to empty array in `getHiddenBlockTypes` selector, but 🤷 . Have you checked about that?

Reply:
That's exactly why I haven't used optional chaining here. As you mentioned, we do always fall back to an empty array:

https://github.com/WordPress/gutenberg/blob/d35f8b0a6be8a87920f6b500015974fda792198d/packages/edit-post/src/store/selectors.js#L232

We can make it safer, but I really think it's not necessary, and it adds unnecessary extra complexity and ambiguity in the future. By not adding it unnecessarily, we save our future selves some time by not having to answer questions like "Why is this optionally chained, perhaps it can be nullish?" or "Was there a time when this could be nullish?" when working with this code.

That's just my five euro cents on this, but I'm happy to add optional chaining here if you feel strongly about it.

Reply:
No strong opinions, just wanted another check 😄